R/detect_mdd.R

```diff
@@ -0,0 +1,47 @@
+################################################################################
+#
+#' @title Determining Minimum Dietary Diversity (MDD)
+#'
+#' @description Identification of individual 6-23 months old children dietary
+#' practices meet the minimum requirement of dietary diversity (at least 5 food
+#' group out of 8 food groups)
+#'
+#' @param age This parameter holds the information about child age in the month
+#'    format, and it should be the continuous variable. It is either generated
+#'    from the child's birth date or self-reported answers from the caregivers.
+#'    If this was generated from the calculation based on the child's date of birth,
+#'    the decimal class of the variable would be the decimal numeric format.
+#'
+#' @param food_score continuous variables indicate child consumed number of food
```

Comment:
State that this is an integer vector, not as a continuous - we don't really use the term continuous in R.